src/nomad_simulations/model_system.py

```diff
@@ -231,25 +254,36 @@ class Cell(GeometricSpace):
         """,
     )
 
+    # topology
+    bond_list = Quantity(
+        type=np.int32,
+        description="""
+        List of index pairs corresponding to (covalent) bonds, e.g., as defined by a force field.
+        """,
+    )  # ? @JosephRudzinski: move position?
+
+    # geometry and analysis
     positions = Quantity(
         type=np.float64,
         shape=['n_cell_points', 3],
         unit='meter',
         description="""
-        Positions of all the atoms in Cartesian coordinates.
+        Positions of all the atoms in absolute, Cartesian coordinates.
```

Comment:
> What is this for exactly?

With a (periodic) cell defined, you could either express the position vectors in terms of the lattice vector basis set, or the basis set in which the lattice vectors were defined (typically spanning a Cartesian space).
We end up going with the latter, since it simplifies large-scale manipulation. I think it's also the most common convention in ab initio.

Reply:
The question was what is the difference between the positions stored here and the positions stored in atomic_cell? And can/should the attribute name here denote that difference (i.e., not just the description)?

Reply:
Ow, yes, I see. Tbh, I think some confusion entered the setup. `Cell` is both the parent to `AtomicCell`, but seems to also contain a grid, i.e. `n_cell_points`. The `positions` and the rest refer to them, though my correction may be wrong.

Let me take a closer look. Chema did push some changes here. I'll get back to you, after merging those.

Reply:
@JosePizarro3 could you pls clarify the meaning of `Cell.n_cell_points`, especially wrt to `AtomicCell`?
I actually added a note for you there when I asked for the review.

Reply:
There is a typo in the description. This should be fixed: 
```python
        description="""
        Positions of
        """,
```

I tried to break the geometric quantities of a cell and the specific atoms sitting in those positions. `Cell` contains thus the `lattice_vectors` and the 0-dimensional points in real space, i.e., `positions` (and also velocities and other info about the Cell), while `AtomsState` has the info of the atom sitting in that position.

So essentially, these are the same both in `Cell` and `AtomicCell`. The only addition of the atomic cell is the atoms state in each of these positions. Whether these `positions` are occupied by entities which are atoms, this depends on the problem. It could be a more complex thing which is centered around them, and for modeling purposes is described like this.

Reply:
> So essentially, these are the same both in Cell and AtomicCell.

Great! That's what I had to know. Then I'll remove `positions` and `velocities` from `AtomicCell`.

Reply:
> There is a typo in the description.

Where?

Reply:
> > There is a typo in the description.
> 
> Where?

Change `atoms` for `points`. And you can rename the quantity above from `n_cell_points` to `n_points`.